src/requestor/arborist.py

```diff
@@ -39,11 +39,34 @@ async def grant_user_access_to_resource(
     }
     await arborist_client.create_resource(parent_path, resource, create_parents=True)
 
+    # Create "reader" and "storage_reader" roles in Arborist.
+    # If they already exist arborist would "Do Nothing"
+    roles = [
+        {
+            "id": "reader",
+            "permissions": [
+                {"id": "reader", "action": {"service": "*", "method": "read"}}
+            ],
+        },
+        {
+            "id": "storage_reader",
+            "permissions": [
+                {
+                    "id": "storage_reader",
+                    "action": {"service": "*", "method": "read-storage"},
+                }
+            ],
+        },
+    ]
+
+    for role in roles:
+        logger.debug(f"Attempting to create role \"{role['id']}\" in Arborist")
+        await arborist_client.create_role(role)
```

Comment:
what do you think about doing something like we do in Fence? https://github.com/uc-cdis/fence/blob/4cfe00b50a802d7379f6f1cb03b00425f7468087/fence/sync/sync_users.py#L1504-L1513
try updating, and if that fails (role doesn't exist), then create

Reply:
so that even if some other policies exist with the same role id, we can overwrite them with ours? Yeah that seems to be a safer option. I can implement it and let you know :)

Reply:
yeah 🤔